packages/cf-style-provider/package.json

```diff
@@ -31,4 +31,4 @@
   "peerDependencies": {
     "react": "^0.14.9 || ^15.0.0-0 || ^16.0.0-0"
   }
-}
+}
```

Comment:
You are going to hate me but new lines at the EOF are a good [thing](https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/). 😆